File: PlutoPilot.py
#pylint: disable=unused-wildcard-import, wildcard-import
from Sensor import Acceleration, Magnetometer, Barometer, Gyroscope
from Estimate import Angle, Velocity
from Control import DesiredAngle
from User import App, Command
from Utils import LED, Graph
from Constants import * 

def plutoInit():
    pass


def onLoopStart():
    LED.flightStatus(DEACTIVATE)
    Error = App.getAppHeading() - Angle.get(AG_YAW)
    arr = []
    x = 10
    y = 15
    test = x if (Error > 10) else y
    print("Error is: ", Error)
    LED.set(RED, ON)

def plutoLoop():
    print("PhoneHeading: ", App.getAppHeading())
    print("Pluto X is at: ", Angle.get(AG_YAW))
    LED.set(RED, ON)
    LED.set(GREEN, ON)
    Command.arm()
    Graph.red(Velocity.get(Z), 1)


def onLoopFinish():
    LED.flightStatus(ACTIVATE)

File: PlutoPilot.cpp
#include <PlutoPilot.h>
#include <Utils.h>
#include <Sensor.h>
#include <Estimate.h>
#include <Control.h>
#include <User.h>
void plutoInit() {
}

void onLoopStart() {
LED.flightStatus(DEACTIVATE);
int16_t Error = (App.getAppHeading() - Angle.get(AG_YAW));
int16_t* arr;
int16_t x = 10;
int16_t y = 15;
int16_t test = (Error > 10) ? x : y;
Monitor.println("Error is: ", Error);
LED.set(RED, ON);
}

void plutoLoop() {
Monitor.println("PhoneHeading: ", App.getAppHeading());
Monitor.println("Pluto X is at: ", Angle.get(AG_YAW));
LED.set(RED, ON);
LED.set(GREEN, ON);
Command.arm();
Graph.red(Velocity.get(Z), 1);
}

void onLoopFinish() {
LED.flightStatus(ACTIVATE);
}

